src/service.rs

```diff
@@ -89,46 +75,64 @@ macro_rules! new_full_start {
             )?;
 
             import_setup = Some((babe_block_import, grandpa_link, babe_link));
-
             Ok(import_queue)
+        // })?
```

Comment:
Is there any reason why we comment this? Is it just to isolate the problem first?

Reply:
Exactly, I commented it out to fix compilation issues, has been re-added now.